cli/stork-publisher-agent/runner.go

```diff
@@ -45,6 +54,48 @@ func NewPublisherAgentRunner[T Signature](
 	}
 }
 
+func (r *PublisherAgentRunner[T]) UpdateBrokerConnections() {
+	r.logger.Info().Msg("Running broker connection updater")
+
+	// query Stork Registry for brokers
+	newBrokerMap, err := r.registryClient.GetBrokersForPublisher(r.config.PublicKey)
+	if err != nil {
+		r.logger.Error().Err(err).Msg("failed to get broker connections from Stork Registry")
+		return
+	}
+
+	// add or update desired connections
+	for brokerUrl, newAssetIdMap := range newBrokerMap {
+		_, exists := r.brokerMap[brokerUrl]
+		if exists {
+			// update connection
+			r.outgoingConnections[brokerUrl].UpdateAssets(newAssetIdMap)
```

Comment:
if the connection already exists, we overwrite the previously configured assets with the current configuration (this is just a cheap in-memory change so we don't check to see if it's changed or not)